docs/development/configuration.md

```diff
@@ -53,6 +52,12 @@ $ node renovate --help
 To configure any `<list>` items, separate with commas.
 E.g. `renovate --labels=renovate,dependency`.
 
+To enable debug logging run:
+
+```sh
+$ LOG_LEVEL=debug node renovate
```

Comment:
node ?

Reply:
It's a copy of the command above. mabe we should use `yarn start` instead 🙃